xml/art_sle_ha_pmremote.xml

```diff
@@ -274,50 +301,213 @@
+  <para>
+   In the following example setup, a physical remote node
+   <systemitem class="domainname">&node3;</systemitem> is used:
+  </para>
   <orderedlist>
    <listitem>
-    <para><xref linkend="sec.ha.pmremote.config.phys.host"/></para>
+    <para><xref linkend="sec.ha.pmremote.config.uc1.virt-guest-nodes"/></para>
    </listitem>
    <listitem>
-    <para><xref linkend="sec.ha.pmremote.config.kvm.guest"/></para>
+    <para><xref linkend="sec.ha.pmremote.config.uc1.kvm.guest"/></para>
    </listitem>
    <listitem>
-    <para><xref linkend="sec.ha.pmremote.integrate.guestsintocluster"/></para>
+    <para><xref linkend="sec.ha.pmremote.config.uc1.integrate"/></para>
    </listitem>
   </orderedlist>
 
-  <note>
-   <title>Use Static Host Names and IP Addresses</title>
-   <para>Add static hosts and IP addresses in <filename>/etc/hosts</filename>.
-   </para>
-  </note>
-
-  <sect2 xml:id="sec.ha.pmremote.config.phys.host">
+  <sect2 xml:id="sec.ha.pmremote.config.uc1.virt-guest-nodes">
    <title>Setting up a Corosync Cluster with Physical Hosts</title>
    <para>
     On the physical host <systemitem class="domainname">&node1;</systemitem>
     proceed as follows:
    </para>
    <procedure>
-    <!--<title>Configuring the Physical Host</title>-->
+      <step>
+       <para>Create a specific authentication key for the &pmremote;
+        service: </para>
+       <screen>&prompt.root;<command>mkdir</command> -p --mode=0755 /etc/pacemaker
+&prompt.root;<command>dd</command> if=/dev/urandom of=/etc/pacemaker/authkey bs=4k count=1</screen>
+       <para>It needs to be synchronized across all cluster nodes with &csync;.
+        The key for the &pmremote; service is different from the cluster
+        authentication key that you create in the &yast; cluster module.</para>
+      </step>
+      <step>
+       <para>In <filename>/etc/csync2/csync2.cfg</filename>, add the
+        following line:</para>
+       <screen>include /etc/pacemaker/authkey</screen>
+      </step>
+      <step>
+       <para>
+        List all cluster and remote nodes in the &csync; configuration file.
+       </para>
+      </step>
+   </procedure>
+  </sect2>
+  <sect2 xml:id="sec.ha.pmremote.config.uc1.kvm.guest">
+   <title>Configuring the KVM Guests</title>
+   <para>
+    The following procedure creates a KVM guest node <systemitem
+      class="domainname">&node4;</systemitem> on your physical remote host
+    <systemitem class="domainname">&node3;</systemitem>:
+   </para>
+   <procedure>
     <step>
-     <para>On two physical hosts, set up a basic two-node cluster as
-      described in <citetitle>&instquick;</citetitle>.
-     <!--<xref linkend="art.ha.install.quick"/>--></para>
+     <para>Create a KVM guest on <systemitem class="domainname"
+      >&node3;</systemitem>. For details refer to the
+      <citetitle>&virtual;</citetitle>, chapter <citetitle>Guest Installation</citetitle>
+      available from <link xlink:href="https://www.suse.com/doc"/>.
```

Comment:
Good point. In use case 1, we would like to install pacemaker_remote directly on a physical machine.